autoload/neomake/makers/ft/sh.vim

```diff
@@ -1,7 +1,7 @@
 " vim: ts=4 sw=4 et
 
 function! neomake#makers#ft#sh#EnabledMakers()
-    return ['shellcheck']
+    return ['shellcheck', 'sh']
```

Comment:
I've thought to have asked already: shouldn't `sh` come first?  (in case you're running the makers successively)